### DISCUSSION TITLE
Creating a Timetable in LaTeX - OSSN

### DISCUSSION THREAD

**Discussion by giscus** [ossnsut/ossnsut.github.io#3] [Ideas]
# Creating a Timetable in LaTeX - OSSN

Writing a cool looking timetable typeset in LaTeX

https://ossnsut.ml/post/latex-timetable/

**Comment by KorigamiK** [comment]
This was pretty fun to make 😄

**Comment by flyingcakes85** [comment]
MS Excel timetables no more XD
